theano/misc/check_blas.py

```diff
@@ -272,6 +272,7 @@ def test():
         GTX 660           2.32s  2.32s
         GTX 580           2.42s         2.47s
         GTX 480           2.87s         2.88s
+        TX1                      7.6s
```

Comment:
isn't TX1 designed to work well with half-precision FP16? it might mislead people that it's slow. anyway